implement_me.py

```diff
@@ -0,0 +1,115 @@
+from dateutil import parser
+from elasticsearch.helpers import bulk
+from elastic_handler import get_elastic_client
+
+INDEX_NAME = 'entry-index'
+ENTRY_TYPE = 'ip_entry'
+MAX_DOCUMENTS = 100000
+BULK_SIZE = 1000
+
+es = get_elastic_client()
+
+
+def clear_index():
+    es.indices.delete(index=INDEX_NAME, ignore=[400, 404])
+    init_mapping()
+
+
+def init_mapping():
+    index_mapping = """{
+        "mappings": {
+            "ip_entry":
+                {
+                  "properties": {
+                    "ip": { 
+                      "type":     "text",
+                      "fielddata": true
+                    }
+                  }
+                }
+        }
+    }"""
+    if not es.indices.exists(INDEX_NAME):
+        es.indices.create(INDEX_NAME, index_mapping)
+
+
+def index(data):
+    """
+    Index the given entries.
+
+    :note: May be called multiple times.
+
+    :param data: A list of 'Entry' instances to index.
+    """
+    indexes = []
```

Comment:
`indexes` is a confusing name.
Did you mean `documents`?

Reply:
yep. fixed